scripts/deploy.ts

```diff
@@ -0,0 +1,140 @@
+import hre from "hardhat";
+const { ethers } = hre;
+
+async function main() {
+  if (hre.network.name === "mainnet") {
+    console.log("\n\n Deploying Contracts to mainnet. Hit ctrl + c to abort");
+  } else if (hre.network.name === "kovan") {
+    console.log("\n\n Deploying Contracts to kovan...");
+  }
+  const INSTA_INDEX = "0x2971AdFa57b20E5a416aE5a708A8655A9c74f723";
```

Comment:
@bhavik-m We need to create a mapping for instaIndex on different chains and then use instaIndex based on network